pkgs/development/compilers/cudatoolkit/common.nix

```diff
@@ -205,6 +221,13 @@ backendStdenv.mkDerivation rec {
       mv pkg/builds/nsight_systems/target-linux-x64 $out/target-linux-x64
       mv pkg/builds/nsight_systems/host-linux-x64 $out/host-linux-x64
     ''}
+      ${lib.optionalString (lib.versionAtLeast version "11.8")
+      # error: auto-patchelf could not satisfy dependency libtiff.so.5 wanted by /nix/store/.......-cudatoolkit-12.0.1/host-linux-x64/Plugins/imageformats/libqtiff.so
+      # we only ship libtiff.so.6, so let's use qt plugins built by Nix.
+      # TODO: don't copy, come up with a symlink-based "merge"
+    ''
+      rsync ${lib.getLib qt6Packages.qtimageformats}/lib/qt-6/plugins/ $out/host-linux-x64/Plugins/ -aP
+    ''}
```

Comment:
I'm not sure I follow the logic of this comment... how does using Qt plugins built by nix alleviate the `.so.5` vs `.so.6` version issue?

Also, why use rsync instead of simply `ln -s`? Ideally we'd like to avoid copying files b/w outputs.

Reply:
> how does using Qt plugins built by nix

...because Nix builds Qt plugins against nixpkgs' libtiff which ships `.so.6`

> Also, why use rsync instead of simply ln -s? Ideally we'd like to avoid copying files b/w outputs.

No good reason at all, other than I wanted a "synchronize these two directory trees recursively" operation, and rsync is one. Either way, we want to be _removing_ all of these plugins, and for all versions of cuda, so we'll have rewritten this patch before we can close https://github.com/NixOS/nixpkgs/issues/224646